lib/i18n/polyfills/numberFormat.js

```diff
@@ -0,0 +1,13 @@
+export const polyfill = () => {
```

Comment:
Just checking, this one doesn't need an `ifNeeded` correct?

Reply:
Same comment as above, I ported this over directly and didn't check the contents with much scrutiny. I will look closer in another PR.